docs/search/index.mjs

```diff
@@ -1,13 +1,95 @@
 import algoliasearch from 'algoliasearch';
 import fs from 'fs/promises';
+import https from 'https';
+
+const retriveAllGitHubDiscussions = () => {
+  const options = {
+    hostname: 'api.github.com',
+    path: '/graphql',
+    method: 'POST',
+    headers: {
+      'Authorization': `Bearer ${
+        process.env['GITHUB_GRAPHQL_KEY']
+      }`,
+      'User-Agent': 'reach',
+    }
+  };
+
+  return new Promise((resolve, reject) => {
+    const request = https.request(options, response => {
+      let data = '';
+
+      response.on('data', chunk => {
+        data += chunk;
+      });
+
+      response.on('end', async () => {
+        const discussions = JSON.parse(
+          data
+        ).data.repository.discussions.edges;
+
+        console.info();
+        console.info('Successfully retrieved discussions');
+        console.info(discussions);
+        console.info();
+
+        resolve(discussions);
+      });
+    }).on('error', (error) => {
+      console.log(error.message);
+      reject(error);
+    });
+
+    const query = `
+    query {
+      repository(name: "reach-lang", owner: "reach-sh") {
+        discussions(first: 100) {
```

Comment:
Why only the first 100? What notion of "first" is this?

Reply:
[8bcb62e](https://github.com/reach-sh/reach-lang/pull/1113/commits/8bcb62ebc00ce96a347069c1f340123825c7c3a2) makes it so that we get all discussions, in ascending order of their creation time.